concordia/static/scss/action-app.scss

```
@@ -186,23 +187,15 @@ main {
     }
 
     .asset-tooltip {
-        width: calc(100% + ($asset-thumbnail-gap * 2));
-        height: calc(100% + ($asset-thumbnail-gap * 2));
-
-        margin: $asset-thumbnail-gap * -1;
         padding: 14px;
-
-        border: $border-width solid $gray-600;
-
-        overflow: hidden;
-
+        border: 2px solid $blue;
         pointer-events: none;
-
         background-color: $white;
-
-        font-size: $concordia-app-font-size-xs;
+        font-size: $concordia-app-font-size-xxs !important;
```

Instead of using `!important`, why not delete the declaration on https://github.com/LibraryOfCongress/concordia/blob/cbe3bc1bbcd91b3fa3261fce306ccf02acee9257/concordia/static/scss/action-app.scss#L284-L286 which is setting `font-size: smaller` on the same element?

```suggestion
        font-size: $concordia-app-font-size-xxs;
```